tests/import_specifications/test_individual_parsers.py

```diff
@@ -806,6 +791,51 @@ def manifest_writer(input_json: dict | list) -> Path:
     return manifest_writer
 
 
+def test_dts_manifest_parse_success(dts_validator: Draft202012Validator):
+    f = _get_test_file("manifest_small.json")
+    res = parse_dts_manifest(f, dts_validator)
+    assert res.results
+    assert res.errors is None
+    assert list(res.results.keys()) == ["gff_metagenome"]
+    assert res.results["gff_metagenome"]
+    assert res.results["gff_metagenome"].source.file == f
+    assert len(res.results["gff_metagenome"].result) == 3
+    for parsed in res.results["gff_metagenome"].result:
+        assert parsed == {"param1": "value1", "param2": "value2"}
```

Comment:
Why did you decide to make all the parameters the same?

Reply:
This was mostly meant as a smoke test / structure test, with more complicated ones elsewhere.
Doesn't matter, though, I updated with more complexity.

Reply:
:+1: